### ISSUE TITLE
Kernel Taint Value 512 Found

### ISSUE THREAD

**Issue by mreed8855** [canonical/checkbox#312]
### Bug Description

Recently I came across the following message for the Kernel taint value:

Kernel taint value is 512
Taint bit value: 9 (kernel issued warning)

The following kernel warning causes this warning:
WARNING: CPU: 64 PID: 1 at arch/x86/events/intel/uncore_discovery.c:184 uncore_insert_box_info+0x134/0x350

The kernel warning message is triggered when SPR MCC is used.

We want to treat this as a known issue and filter the logs for the message and ignore it in the future.

### To Reproduce

1.  Run the kernel Taint test on a system with a SPR MCC CPU

### Environment

Jammy 5.15
A system using an Sapphire Rapids MCC CPU

### Relevant log output

```shell
Partial Dmesg log

Jan  4 07:49:51 proven-gnu kernel: [    4.202465] WARNING: CPU: 64 PID: 1 at arch/x86/events/intel/uncore_discovery.c:184 uncore_insert_box_info+0x134/0x350
Jan  4 07:49:51 proven-gnu kernel: [    4.202474] Modules linked in:
Jan  4 07:49:51 proven-gnu kernel: [    4.202478] CPU: 64 PID: 1 Comm: swapper/0 Not tainted 5.15.0-56-generic #62-Ubuntu
Jan  4 07:49:51 proven-gnu kernel: [    4.202482] Hardware name: Dell Inc. PowerEdge T560/0PWDKY, BIOS 0.2.13 12/09/2022
Jan  4 07:49:51 proven-gnu kernel: [    4.202485] RIP: 0010:uncore_insert_box_info+0x134/0x350
Jan  4 07:49:51 proven-gnu kernel: [    4.202488] Code: c2 01 48 83 c0 04 39 d1 0f 8e c6 01 00 00 49 8b 4c 24 38 8b 0c 01 41 89 0c 07 49 8b 74 24 40 8b 34 06 41 89 34 06 39 f9 75 cf <0f> 0b 4c 89 ff e8 52 86 32 00 4c 89 f7 e8 4a 86 32 00 5b 41 5c 41
Jan  4 07:49:51 proven-gnu kernel: [    4.202494] RSP: 0000:ff58c947c01efc98 EFLAGS: 00010246
Jan  4 07:49:51 proven-gnu kernel: [    4.202498] RAX: 0000000000000008 RBX: 0000000000000000 RCX: 0000000000000003
Jan  4 07:49:51 proven-gnu kernel: [    4.202500] RDX: 0000000000000002 RSI: 0000000000018000 RDI: 0000000000000003
Jan  4 07:49:51 proven-gnu kernel: [    4.202503] RBP: ff58c947c01efcc0 R08: 0000000000000010 R09: ff356d890b9282f0
Jan  4 07:49:51 proven-gnu kernel: [    4.202505] R10: ff356d988a2e2000 R11: ffffffffffffe000 R12: ff356d9889edc6c0
Jan  4 07:49:51 proven-gnu kernel: [    4.202507] R13: ff58c947c01efcf8 R14: ff356d890b928db0 R15: ff356d890b9282f0
Jan  4 07:49:51 proven-gnu kernel: [    4.202510] FS:  0000000000000000(0000) GS:ff356d983fa00000(0000) knlGS:0000000000000000
Jan  4 07:49:51 proven-gnu kernel: [    4.202513] CS:  0010 DS: 0000 ES: 0000 CR0: 0000000080050033
Jan  4 07:49:51 proven-gnu kernel: [    4.202515] CR2: 0000000000000000 CR3: 00000010be610001 CR4: 0000000000771ee0
Jan  4 07:49:51 proven-gnu kernel: [    4.202518] DR0: 0000000000000000 DR1: 0000000000000000 DR2: 0000000000000000
Jan  4 07:49:51 proven-gnu kernel: [    4.202520] DR3: 0000000000000000 DR6: 00000000fffe07f0 DR7: 0000000000000400
Jan  4 07:49:51 proven-gnu kernel: [    4.202522] PKRU: 55555554
Jan  4 07:49:51 proven-gnu kernel: [    4.202524] Call Trace:
Jan  4 07:49:51 proven-gnu kernel: [    4.202526]  <TASK>
Jan  4 07:49:51 proven-gnu kernel: [    4.202530]  parse_discovery_table.isra.0+0x162/0x1a0
Jan  4 07:49:51 proven-gnu kernel: [    4.202534]  intel_uncore_has_discovery_tables+0x19e/0x270
Jan  4 07:49:51 proven-gnu kernel: [    4.202538]  ? type_pmu_register+0x16/0x42
Jan  4 07:49:51 proven-gnu kernel: [    4.202545]  intel_uncore_init+0xe3/0x226
Jan  4 07:49:51 proven-gnu kernel: [    4.202549]  ? type_pmu_register+0x42/0x42
Jan  4 07:49:51 proven-gnu kernel: [    4.202553]  do_one_initcall+0x46/0x1e0
Jan  4 07:49:51 proven-gnu kernel: [    4.202559]  do_initcalls+0x12f/0x159
Jan  4 07:49:51 proven-gnu kernel: [    4.202564]  kernel_init_freeable+0x162/0x1b5
Jan  4 07:49:51 proven-gnu kernel: [    4.202568]  ? rest_init+0x100/0x100
Jan  4 07:49:51 proven-gnu kernel: [    4.202575]  kernel_init+0x1b/0x150
Jan  4 07:49:51 proven-gnu kernel: [    4.202578]  ? rest_init+0x100/0x100
Jan  4 07:49:51 proven-gnu kernel: [    4.202581]  ret_from_fork+0x1f/0x30
Jan  4 07:49:51 proven-gnu kernel: [    4.202586]  </TASK>
Jan  4 07:49:51 proven-gnu kernel: [    4.202588] ---[ end trace 8ed67df079f38d94 ]---
```

**Comment by mreed8855** [comment]
This is a kernel issue and there is a launchpad bug tracking this so I am going to close it

**Comment by mreed8855** [comment]
The launchpad bug that closed this issue is https://bugs.launchpad.net/ubuntu/+source/linux-hwe-5.15/+bug/2008037
